omigami/__init__.py

```diff
@@ -8,10 +8,12 @@
 
 from pathlib import Path
 
+from .plotting import MoleculePlotter, SpectraComparisonPlotter
 from ._version import get_versions
+from .utilities import GnpsHelper
+from .utilities import SpectrumDataFrameHelper
```

Comment:
True. GNPS one should be to fetch data from gnps endpoints. The dataframe one should be to a) convert from one to the other type and b) to provide useful dataframe operations. Since we don't control either Spectrum nor DataFrame classes we couldn't just place them methods inside them (at least without a wrapper class, but I am against it because names would be confusing [having 2 "Spectrum" classes or "SpectrumWrapper" or whatever would be not nice]). I can write something like this in a docstring inside each class. Do you think we should  change the names or repurpose these classes?

Reply:
Okay, so I placed the function in GNPSHelper inside SpectrumDataFrameHelper as its purpose is to get a dataframe from a GNPS id using GNPS API.
I deleted GNPSHelper.